servers/navigation/navigation_mesh_generator/navigation_mesh_generator.h

```diff
@@ -32,10 +32,11 @@
 #define NAVIGATION_MESH_GENERATOR_H
 
 #ifndef _3D_DISABLED
```

Comment:
Do we need to disable the mesh generator is 3D is disabled?  `NavigationMesh` is still used in 2D no?